Changelog.md

```diff
@@ -8,6 +8,7 @@ Compiler Features:
 
 
 Bugfixes:
+ * SMTChecker: Fixed crash when used with fixed-sized arrays.
```

Comment:
The bug itself was somewhere else, but I don't know if we want to mention that here.